src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1848,6 +1877,17 @@ namespace DataZoneEquipment {
         DataHVACGlobals::MinAirLoopIterationsAfterFirst = minIterations;
     }
 
+    Real64 EquipList::SequentialHeatingFraction(const int equipNum)
+    {
+        return ScheduleManager::GetCurrentScheduleValue(SequentialHeatingFractionSchedPtr(equipNum));
+    }
+
+    Real64 EquipList::SequentialCoolingFraction(const int equipNum)
+    {
+        return ScheduleManager::GetCurrentScheduleValue(SequentialCoolingFractionSchedPtr(equipNum));
+    }
```

Comment:
@nmerket @Myoldmopar This is all ready to go in except for the small performance degradation.

Performance Test Time | 339774256113 | 339348886410 | -0.1253487841083034%
-- | -- | -- | --

For the default case where `SequentialCooling/HeatignFraction` was a simple value of 1.0, it's now a 2-layer function call out to `GetCurrentScheduleValue`.
Too small to worry about? 
Let the schedule manager refactor recover some of the performance?
Split the difference and put an if in these functions to return 1.0 if the `...SchedPtr = ScheduleAlwaysOn`?
Other?

Reply:
We can live with the 0.1% performance degradation. My bet is that the slow down comes from looking looking up schedule values instead of reading a single value off a struct. I'd be surprised if the extra layer of function call makes that much of a difference.

Reply:
A 0.1% performance degradation may not seem that important, but we are looking at trying to scrape every bit of performance we can, and small steps backwards can add up.

That said, I'm not sure where we get if we add the IF logic in there to see if it is always on.  We may still have a performance degradation of like 0.05%.

In this particular case, I'd say let it go in because of the schedule manager refactor work.